staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -128,9 +128,8 @@ func (r *rudimentaryErrorBackoff) OnError(error) {
 	r.lastErrorTimeLock.Lock()
 	defer r.lastErrorTimeLock.Unlock()
 	d := time.Since(r.lastErrorTime)
-	if d < r.minPeriod && d >= 0 {
+	if d < r.minPeriod {
```

Comment:
did the time.Since semantics change?

Reply:
https://golang.org/pkg/time/#hdr-Monotonic_Clocks

Reply:
golang 1.9